src/wp-admin/includes/file.php

```diff
@@ -635,7 +635,7 @@ function wp_edit_theme_plugin_file( $args ) {
 			wp_opcache_invalidate( $real_file, true );
 
 			if ( ! isset( $result['message'] ) ) {
-				$message = __( 'Something went wrong.' );
+				$message = __( 'An unexpected error occurred. Please try again later or contact support.' );
```

Comment:
Errors are by definition unexpected so no need to include the word.

As this error is shown to admins, they may not have a support person to contact.

:1234: This applies elsewhere in the pull request where the phrase "unexpected error" is used.

```suggestion
				$message = __( 'An error occurred. Please try again later.' );
```